packages/darkice/darkice@.service

```diff
@@ -0,0 +1,10 @@
+[Unit]
+Description=DarkIce Live Stream %i
+After=network.target
+
+[Service]
```

Comment:
Since this is a system service maybe it should be run under a custom user, like `darkice`, for security reasons?

Reply:
Probably it would make more sense to turn this into a user service template?

Something like this (see 22f1902):

```ini
[Unit]
Description=DarkIce Live Stream %i
Documentation=man:darkice(1)
After=network.target

[Service]
Type=simple
ExecStart=/usr/bin/darkice -c %h/.config/darkice/%i.conf

[Install]
WantedBy=default.target
```

I'm not sure whether this is correct, though.

Reply:
Right, I somehow thought darkice was the streaming _server_. In that case it would have made sense to make it a system service running under a `darkice` user in the background. But since it is a source client, there are a few points to consider:
- it should probably run under a regular user during a user session
- are there use cases where you would like to _enable_ the service, i.e. start it on each login?
- are there use cases where you would run darkice as a system service? Like a radio station using a headless system for streaming and everything else happens before the audio input?

Maybe if we can't answer some of these questions it would be ok to keep the current service and wait for people to come here with their own use cases 😂